illustrations/assets/index.md

```diff
@@ -7,143 +7,142 @@ layout: app
 
 <hr />
 
-<div class="u-clearfix">
+<div class="container">
 	<h2 class="m-bottom-2">Spot Illustration</h2>
-	<div class="g-1_4">
-		<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/laptop-light@2x.png" alt="Dreamhost Spot Illustration" />
-		<p class="p-2 bg-c-g100 t-center"><strong>Laptop — Light</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/laptop-light@2x.png" download>PNG</a></p>
-	</div>
-	<div class="g-1_4">
-		<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/laptop-dark@2x.png" alt="Dreamhost Spot Illustration" />
-		<p class="p-2 bg-c-g100 t-center"><strong>Laptop — Dark</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/laptop-dark@2x.png" download>PNG</a></p>
-	</div>
-	<div class="g-1_4">
-		<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-domain@2x.png" alt="Dreamhost Spot Illustration" />
-		<p class="p-2 bg-c-g100 t-center"><strong>Domain</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-domain@2x.png" download>PNG</a></p>
-	</div>
-	<div class="g-1_4">
-		<img class="m-bottom-0 p-bottom-0" src="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-finance@2x.png" alt="Dreamhost Spot Illustration" />
-		<p class="p-2 bg-c-g100 t-center"><strong>Finance</strong><br /> Format: <a href="{{site.baseurl}}/assets/images/illustration/resource/spot/v2/2x/spot_illo-finance@2x.png" download>PNG</a></p>
+	<div class="row">
```

Comment:
The text gets lengthy and these illustrations have more detail that might look funky when squished together. I would say remove the `sm` columns and just make them go full width below 750px.